pegasus/migrations/137_rename_user_storage_ids.rb

```diff
@@ -5,22 +5,29 @@
 
     unless [:production].include?(rack_env) || ENV['CI']
```

Comment:
Do we need to remove the `CI` check here too?

Reply:
Yes! Good catch